### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by boomski**
Added Fubo Cycling

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #EXTM3U x-tvg-url="http://epg.streamstv.me/epg/guide-usa.xml.gz"
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/HmqRMIq.png" group-title="Sport",Fubo Cycling
+http://ummodigital.com:8000/jorge1/jorge1/3426
```
Comment (freearhey) [inline channels/us.m3u]: This channel does not work for me.

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by boomski** [comment]
test

**Comment by freearhey** [comment]
Honestly, I still do not understand why it works for you but not for me. Just in case, I checked the link through Postman and it also returns me 401 error.

**Comment by freearhey** [comment]
@boomski can you remove the link `http://ummodigital.com:8000/jorge1/jorge1/3426` from PR so that I can approve it and keep a new logo for the FOX Business Network

**Comment by boomski** [comment]
@freearhey  Link removed

**Review by freearhey — APPROVED** [review]

**Pull request merged**